src/app/containers/CurrentCourthouseDisplay.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { RootState } from '../store';
+import {
+    Courthouse,
+    IdType
+} from '../api/Api';
+import { courthouseById } from '../modules/courthouse/selectors';
+import { currentCourthouse } from '../modules/user/selectors';
+import { Button } from 'react-bootstrap';
+import { Glyphicon } from 'react-bootstrap';
+
+interface CurrentCourthouseDisplayStateProps {
+    courthouse?: Courthouse;
+}
+
+interface CurrentCourthouseDisplayDispatchProps {
+}
+
+interface CurrentCourthouseDisplayProps {
+    id?: IdType;
+}
+
+class CurrentCourthouseDisplay extends React.PureComponent<
+    CurrentCourthouseDisplayProps & CurrentCourthouseDisplayStateProps & CurrentCourthouseDisplayDispatchProps> {
+
+    render() {
+        const { courthouse } = this.props;
+        return (
+            <span>
+                {courthouse ? courthouse.name : 'not selected'}
+                <Button bsStyle="link" onClick={() => window.location.reload()}>
```

Comment:
I think here if we set the courthouse to nothing using the action that you made `updateCurrentCourthouse`.  The state would get updated and it would take you back to the home page with the selector.  The only problem is that right now I don't think we're handling the change of courthouse well enough (i.e. re-fetching resources on change etc).  So I think this is good for now, but in the near future we may need to add some '[side effects](https://goshakkk.name/redux-side-effect-approaches/)' to whenever the courthouse changes.

Reply:
I was able to add some additional fetch actions to the updateCurrentCourthouse action, so instead of having this refresh button I have included the current courthouse selector in the nav bar